examples/MerminMagic/main.jl

```diff
@@ -1,85 +1,81 @@
 # Mermin-Peres Magic Square Game Algorithm
 # https://www.gregegan.net/SCIENCE/MPMS/MPMS.html
 using Yao
-using Yao.EasyBuild, YaoPlots
-using Random
+
+# measurement operator table
+# see Table S1 in https://arxiv.org/abs/2206.12042
+op_mtx = [kron(I2, Z) kron(Z, I2) kron(Z, Z); kron(X, I2) kron(I2, X) kron(X, X); kron(-X, Z) kron(-Z, X) kron(Y, Y)]
+
+"""
+    bell_state(which::Int)::AbstractRegister
+
+Initialize bell state on a quantum register.
+
+# Arguments
+- 'which::Int': which of the four bell states to initialize
+  1 corresponds to ``(|00> + |11>)/√2``
+  2 corresponds to ``(|00> - |11>)/√2``
+  3 corresponds to ``(|01> + |10>)/√2``
+  4 corresponds to ``(|01> - |10>)/√2``
+"""
+function bell_state(which::Int)
+
+    @assert 1 <= which <= 4 "Input $(which) is not in range 1..4"
+    reg = zero_state(2)
+    apply!(reg, chain(put(2, 1 => H), cnot(2, 1, 2)))
+    (which == 2 || which == 4) && apply!(reg, put(2, 1 => Z))
+    (which == 3 || which == 4) && apply!(reg, put(2, 1 => X))
+    return reg
+end
 
 """
-    init_double_bell_state(reg::ArrayReg)
+    init_double_bell_state()::AbstractRegister
```

Comment:
```suggestion
    init_double_bell_state() -> AbstractRegister
```
In docstring, we use arrows.

Please check the style guide: https://github.com/invenia/BlueStyle#documentation